src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-sentry_sdk.py

```diff
@@ -9,6 +9,7 @@
 #
 # SPDX-License-Identifier: GPL-2.0-or-later
 # ------------------------------------------------------------------
+import sentry_sdk.integrations as si
```

Comment:
Whenever you import a 3rd party library in a hook you should put it inside an `exec_statement()` ([docs](https://pyinstaller.readthedocs.io/en/stable/hooks.html#useful-items-in-pyinstaller-utils-hooks)). That would mean moving this down to join the code at the bottom which should go inside the exec-ed statement too.

Reply:
The problem is, if I'm correct, that `exec_statement()` communicates via prints. If I print the list, I would have to convert it back to list to extend the `hiddenimports` list which would add a lot of overhead. I would have to encode the list as json, print the json and then to parse the json for getting the original list.

Reply:
That list is tiny. I get 14 microseconds of overhead...